test/helpers/node.go

```diff
@@ -31,7 +31,8 @@ import (
 
 var (
 	//SSHMetaLogs is a buffer where all commands sent over ssh are saved.
-	SSHMetaLogs = NewWriter(new(bytes.Buffer))
+	SSHMetaLogs      = NewWriter(new(bytes.Buffer))
+	sendToLog   bool = true // Set to false if commands don't need to save in the logs
```

Comment:
The reason why I did not use a variadic argument is because I think that a great improvement to Ginkgo Exec will have the same approach as we have in `helpers.Curlfail` where you can run something like this: 

```
kub.Exec("kubectl -n %s get pods", namespace)
```

That will clean a lot of `fmt.Sprintf` that are useless nowadays and will make the code simpler.

Reply:
@eloycoto your comment does not make sense to me. What does this accomplish that using a variadic argument does not?

Reply:
You can only have one variadic argument type, so it'll be a string with all the options that'll passed to `Exec`, so if you add a new argument here, you will have something: 

`Exec("kubectl -n %s", true, "kube-system")`

Reply:
My suggestion of having an `Option` type like https://github.com/cilium/cilium/blob/9f63c3798c562f17037e15e58cb501fea9f2ce98/vendor/github.com/coreos/etcd/clientv3/kv.go#L46 solves that problem.

Reply:
The problem is the same no? then, you will do some function before you run Exec. 

Does not matter, I'll change to variadict argument and keep the `Sprintf`